File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Addresses/IAddressData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Addresses
{
    public interface IAddressData :  CoreObject.IObjectID,CoreObject.IObjectTitle, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IAddressFullAddress, IAddressPostCode, IAddressCountry, IAddressCity
    {

    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Addresses/IAddressData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Addresses;
use Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IAddressData extends IAddressFullAddress,IAddressCity,IAddressCountry,IAddressPostCode,  CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{


}